src/components/inputs/TextField.js

```diff
@@ -351,7 +347,7 @@ export default class TextField extends BaseInput {
               }),
               color: floatingPlaceholderState.interpolate({
                 inputRange: [0, 1],
-                outputRange: [placeholderColor, this.getStateColor(floatingPlaceholderColor)]
+                outputRange: [placeholderColor, this.getStateColor(floatingPlaceholderColor || {focus: Colors.blue30, default: Colors.grey30})]
```

Comment:
why to set all DEFAULT_COLOR_BY_STATE colors to 'grey10' if these are the defaults you want? Why to hard-code them here (and in line #368)?

Reply:
That was part of the bug with the colors. 

The component assumed we have the same color states for placeholder and the input text color, but they're not the same according to the guidelines. 
The color states for the input text is different than the color states of the placeholder 

So basically, as I wrote in the previous comment `DEFAULT_COLOR_BY_STATE` represent the color states for the input text
and I pass a custom object for each part (placeholder, underline, etc..) according to our guidelines.